recipes/asmjit/all/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
-cmake_minimum_required(VERSION 3.1)
-project(cmake_wrapper)
+cmake_minimum_required(VERSION 3.8)
```

Comment:
doesn't really matter but why 3.8 now?

Reply:
I made the change when I was modifying the recipe and forgot to change it back.
I will revert it back.